src/Tests/dotnet.Tests/CommandTests/ToolUpdateLocalCommandTests.cs

```diff
@@ -123,6 +142,28 @@ public void WhenRunWithPackageIdItShouldUpdateFromManifestFile()
             AssertUpdateSuccess();
         }
 
+        [Fact]
+        public void WhenRunWithUpdateAllItShouldUpdateFromManifestFile()
+        {
+            _toolRestoreCommand.Execute();
+            new ToolRestoreCommand(
+                Parser.Instance.Parse($"dotnet tool update {_packageIdB.ToString()} --local"),
```

Comment:
It looks like this is passing the parse result of a tool update command to a `ToolRestoreCommand`.  I think this would be better:

```suggestion
                Parser.Instance.Parse($"dotnet tool restore"),
```